### PR TITLE
fix(sdk): Improve filter/summarize/notebook layouts

### PR DESCRIPTION
**Pull request by oisincoveney**
Just a few adjustments to the filter, summarize, and notebook components. I've added a few tweaks that ensure that the components fill their frames, so there isn't any weirdness when using the components in a custom context. I've added scrolling, 100% height and width, and a few other things

### CODE REVIEW

File: frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeContent/SummarizeContent.tsx
Comment (oisincoveney) [inline frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeContent/SummarizeContent.tsx:1]: I split the component into the `SummarizeBreakoutColumnList` and `SummarizeAggregationItemList` so we could have a little more granularity when adding the divider, padding, and other things.

File: frontend/src/metabase/querying/components/FilterContent/FilterSearchInput/FilterSearchInput.tsx [hunk 1]
```diff
@@ -24,15 +24,13 @@ export function FilterSearchInput({
   );
 
   return (
-    <Flex mx="md" justify="end" style={{ flex: 1 }}>
```
Comment (oisincoveney) [inline frontend/src/metabase/querying/components/FilterContent/FilterSearchInput/FilterSearchInput.tsx:27]: Moved this from this component (whose job should _only_ be is to handle text for the search) and moved it to the `FilterModal` component. We can then have a different layout for the `Filter` SDK component

File: enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/Filter.tsx [hunk 2]
```diff
@@ -51,18 +51,22 @@ const FilterInner = ({
...
-        searching={isSearching}
-        onChange={handleChange}
-        onInput={handleInput}
-        onTabChange={setTab}
-      />
+    <Stack w="100%" h="100%">
+      <Group position="right">
+        <FilterContent.Header value={searchText} onChange={handleSearch} />
+      </Group>
+      <Box h="100%" style={{ flex: 1, overflow: "hidden" }}>
```
[the remark this answers is not in the document]
Comment (heypoom) [reply enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/Filter.tsx:58]: +1 for CS* styles

**Review by heypoom — APPROVED** [review]
Nice!

**Pull request merged**